### ISSUE TITLE
sqlite3.OperationalError: unable to open database file

### ISSUE THREAD

**Issue by summertoyou** [jay0lee/got-your-back#97]
The issue tracker is for reporting product deficiencies. How do I questions should be posted to the discussion forum at https://groups.google.com/group/got-your-back. When in doubt, start at the discussion forum and return here only when instructed to do so.

Please confirm the following:
* I have upgraded to the latest GYB release from https://github.com/jay0lee/got-your-back/releases and I still have this issue.
* I am typing the command as described in the GAM Wiki at https://github.com/jay0lee/got-your-back/wiki

Full steps to reproduce the issue:
1. My-MacBook-Pro:got-your-back-0.45 S$ python3 gyb.py --email myemail@myemail.com --action restore-mbox --local-folder /Users/MyUser/my_gmail_backup.mbox
2.
3.

Expected outcome (what are you trying to do?): Restored Successfully from the mbox file

Actual outcome (what errors or bad behavior do you see instead?): 
Using backup folder /Users/S/my_gmail_backup.mbox
Traceback (most recent call last):
  File "gyb.py", line 1503, in <module>
    main(sys.argv[1:])
  File "gyb.py", line 925, in main
    detect_types=sqlite3.PARSE_DECLTYPES)
sqlite3.OperationalError: unable to open database file

**Comment by jay0lee** [comment]
--local_folder parameter should indicate a FOLDER, not a file. Either change --local_folder to point to /Users/MyUser/ or move my_gmail_backup.mbox into it's own sub-folder and specify that folder. GYB will scan the specified folder and any sub-folders for files ending in .mbox or .mbx and use those for restore.

**Comment by summertoyou** [comment]
Using backup folder /Users/S

Restoring from 690.59MB file /Users/S/my_gmail_backup.mbox...
large files may take some time to open.

Traceback (most recent call last):
  File "gyb.py", line 1503, in <module>
    main(sys.argv[1:])
  File "gyb.py", line 1208, in main
    labels = labels.decode('string-escape')
**AttributeError: 'str' object has no attribute 'decode'**

It seemed like it was going to work then I got this error.
Something I did wrong?
